.github/workflows/format.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Install clang-format
         run: |
           sudo apt-get update
-          sudo apt-get -y install clang-format-12
+          sudo apt-get -y install clang-format
```

Comment:
It might not be a bad idea to keep using a certain version explicitly, to avoid different versions potentially behaving differently, so this would be `clang-format-14` for example.